Tests/configure_and_test_integration_instances.py

```diff
@@ -705,12 +749,11 @@ def set_marketplace_gcp_bucket_for_build(client, prints_manager, branch_name, ci
     server_configuration = {
         'content.pack.verify': 'false',
         'marketplace.initial.sync.delay': '0',
-        'content.pack.ignore.missing.warnings.contentpack': 'true',
```

Comment:
I don't see changes related to `Prepare Content Packs For Testing` step. Lets discuss it tomorrow.